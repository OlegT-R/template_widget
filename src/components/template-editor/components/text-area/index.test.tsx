import React from 'react';

import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { TextArea } from '.';

const textArea = {
  id: 'textArea',
  value: 'textArea',
};

const onSetActiveTextArea = () => null;

test('<TextArea />', () => {
  const component: any = shallow(
    <TextArea
      node={textArea}
      onSetActiveTextArea={onSetActiveTextArea}
      parentNode={[textArea]}
    />
  );
  expect(toJson(component)).toMatchSnapshot();
});
