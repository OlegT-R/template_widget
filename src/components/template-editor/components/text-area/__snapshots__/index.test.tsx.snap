// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextArea /> 1`] = `
<TextArea
  autosize={
    Object {
      "minRows": 2,
    }
  }
  onBlur={[Function]}
  onChange={[Function]}
  value="textArea"
/>
`;
